.changeset/itchy-sloths-return.md

```diff
@@ -0,0 +1,5 @@
+---
+'wingman-fe': minor
+---
+
+Use sku 12
```

Comment:
This is also technically inconsequential because it's a dev dependency, but I assume we should do a release anyhow just to surface any issues